publish/package.json

```diff
@@ -27,6 +27,7 @@
     "@angular/cli": "6.0.7",
     "classlist.js": "^1.1.20150312",
     "ids-enterprise": "^4.8.0-dev.20180621",
-    "jquery": "3.3.1"
+    "jquery": "3.3.1",
+    "@types/jquery": "~3.2.16"
```

Comment:
Should we use the same version here? https://www.npmjs.com/package/@types/jquery/v/3.3.1

Reply:
I think yes it makes sense to keep the typing package version to be the same as the `jquery` version. Will change